brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-761 Improvement : Global block changes - Acceptance Testing Round 2 Review Changes.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-761

### Screenshots

- Consistent unit PX/EM where needed ( Heading, Blockquotes, CTA, carousel, grid, masonry, TOC )
- Font weight should be Default when font family is changed - https://share.getcloudapp.com/lluEB92Q
- The first tab from General & Style will be open on the first drag and drop - https://share.getcloudapp.com/z8uLNPQx
- Toggles need to be aligned properly to the end. - https://share.getcloudapp.com/JruOd19X 

### Types of changes




- Improvements.

### How has this been tested?

- Check each block. No single unit is present for each option, the Unit should be consistent with PX/EM.
- Check Font weight should be Default when the font family is changed. Please check the screencast. Check editor frontend also. Font family should be applied in the editor as well as on the front. check by changing family save, refresh, and check-in editor it should be the same applied to a specific field.
- Drag and drop any block and the first tab from each General and Style should be open on the first drag and drop.
- Check toggle is aligned properly at the end or not.

### Checklist:
